src/components/MultiHopTrade/components/TradeConfirm/components/ExpandedStepperSteps.tsx

```diff
@@ -194,6 +204,55 @@ export const ExpandedStepperSteps = ({ activeTradeQuote }: ExpandedStepperStepsP
   )
   const prevFirstHopAmountCryptoBaseUnit = usePrevious(firstHopAmountCryptoBaseUnit)
 
+  const swapper = swappers[activeTradeQuote.swapperName]
+
+  const { data: firstHopSellTradeStatus } = useQuery({
+    queryKey: tradeStatusQueryKey(activeSwapId ?? '', firstHopSwap.sellTxHash ?? ''),
+    queryFn:
+      swapper && firstHopSwap.sellTxHash && activeSwapId && firstHopSellAccountId
+        ? () =>
+            fetchTradeStatus({
+              swapper,
+              sellTxHash: firstHopSwap.sellTxHash ?? '',
+              sellAssetChainId: fromAccountId(firstHopSellAccountId).chainId,
+              address: fromAccountId(firstHopSellAccountId).account,
+              swap: activeSwap,
+              stepIndex: 0,
+              config: getConfig(),
+            })
+        : skipToken,
+    staleTime: TRADE_STATUS_POLL_INTERVAL_MILLISECONDS,
+    gcTime: TRADE_STATUS_POLL_INTERVAL_MILLISECONDS,
+  })
+
+  const { data: lastHopSellTradeStatus } = useQuery({
+    queryKey: tradeStatusQueryKey(activeSwapId ?? '', lastHopSwap.sellTxHash ?? ''),
+    queryFn:
+      swapper && lastHopSwap.sellTxHash && activeSwapId && lastHopSellAccountId
+        ? () =>
+            fetchTradeStatus({
+              swapper,
+              sellTxHash: lastHopSwap.sellTxHash ?? '',
+              sellAssetChainId: fromAccountId(lastHopSellAccountId).chainId,
+              address: fromAccountId(lastHopSellAccountId).account,
+              swap: activeSwap,
+              stepIndex: 0,
+              config: getConfig(),
+            })
+        : skipToken,
+
+    staleTime: TRADE_STATUS_POLL_INTERVAL_MILLISECONDS,
+    gcTime: TRADE_STATUS_POLL_INTERVAL_MILLISECONDS,
+  })
```

Comment:
I didn't want to create a hook for this, seems reusable enough to use them, the only weird thing is the staleTime and gcTime but as it's shared using a constant with `tradeExecution`, we should be good to go!

Confirmed in the jam that only 1 query is sent every 5s, shared between tradeExecution and thise screen!

Reply:
Sound sane to me, don't think we necessarily need to abstract yet, since there is only one usage as `useQuery()` (the current others leverage `qc.fetchQuery()`